packages/openapi-v3/src/__tests__/unit/decorators/param/param.decorator.unit.ts

```diff
@@ -32,10 +32,11 @@ describe('Routing metadata for parameters', () => {
 
       const expectedSpec = anOperationSpec()
         .withOperationName('greet')
+        .withControllerName('MyController')
         .withParameter(paramSpec)
         .withResponse(200, {description: 'Return value of MyController.greet'})
         .build();
-      expect(actualSpec.paths['/greet']['get']).to.eql(expectedSpec);
+      expect(actualSpec.paths['/greet']['get']).to.containEql(expectedSpec);
```

Comment:
nitpick: could you update the `withOperationName` function as well to generate the `operationId` so that the `expectedSpec` contains that field?
The `.eql()` is used intended to make sure the generated spec and `expectedSpec` are exactly the same. Thanks.

Reply:
Fixed.